source/making_payments/index.html.md.erb

```diff
@@ -77,37 +100,36 @@ datastore mapped to the payment ID just after you create a payment.
  * not paid because of a technical error
 
 Your service should use the API to check the payment status when the user
-reaches the return URL, and provide an appropriate response based on the final
+reaches the `return_url`, and provide an appropriate response based on the final
 status of the payment attempt.
 
 ## When a user does not complete their payment journey
 
-The user may close their browser or lose internet connection in the middle of
-the payment flow on GOV.UK Pay. These users will not be redirected back to
-your service.
-
-You can still check on the status of these payments by making a GET request
-using the Location Header or Self Link, the same way you would if they were
-redirected, but just after a set time (eg, two hours).
+Users may close their browser or lose internet connection in the middle of
+their payment journey on GOV.UK Pay. In this event, users will not be
+redirected back to your service.
 
->Note: GOV.UK Pay will expire incomplete payments after 90 minutes, but you should expect an occasional success or failure if the user experienced problems right at the moment of the redirect.
+You can still [check the status of these
+payments](/payment_flow/#check-the-status-of-a-payment), but GOV.UK Pay will
+expire incomplete payments after 90 minutes so you should only check after
+this amount of time.
```

Comment:
It’s fine for them to check the payment status before the expiry time but they need to be aware that it’s not necessarily final until it’s either completed or expired. Maybe something like this?

“You can still [check the status of these payments](/payment_flow/#check-the-status-of-a-payment), but be aware that GOV.UK Pay allows 90 minutes for a user to complete their payment before expiring it, so you can only make a final decision about whether or not a payment was successful after this time.”

Reply:
Tried to alter language